src/ros_video_components/src/ros_driving_mode_switching.cpp

```diff
@@ -0,0 +1,86 @@
+#include "ros_video_components/ros_driving_mode_switching.hpp"
+
+#define RECT_X 0
```

Comment:
These `#defines` are not being used anywhere, are they? I reckon get rid of all of these unnecessary `#define`s